src/content/development/security/containers/_index.en.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Container Requirements"
+weight: 50
+---
+
+Current privilege setup is as follows, for non-test containers deployed by Submariner.
+Production containers not described here don’t use extra capabilities.
```

Comment:
```suggestion
The current privilege requirements for non-test containers deployed by Submariner are described below.
Any other production containers not described here don’t require extra privileges.
```

Reply:
As I tried to explain in the commit message, this PR describes the current settings, it doesn’t describe the requirements (which are less than the current settings).